policyengine_us/variables/gov/states/nc/ncdhhs/scca/nc_scca_age_group.py

```diff
@@ -14,27 +14,23 @@ class nc_scca_age_group(Variable):
     def formula(person, period, parameters):
         """
         Returns the age group for NC SCCA:
-        1 - Infant (0 to <=1)
+        1 - Infant (0 to <1)
         2 - Toddler (1 to <3)
         3 - Preschooler (3 to <6)
-        4 - School age (6+)
-        """
 
-        p = parameters(period).gov.states.nc.ncdhhs.scca
+        The school age group is hardcoded as 4 (one more than the maximum value in the parameter)
+        """
         age = person("age", period)
-        is_disabled_age = person(
-            "nc_scca_is_eligible_disabled_age", period
-        )  # disabled & < 18
+        is_school_age = person("nc_scca_is_school_age", period)
+        p = parameters(period).gov.states.nc.ncdhhs.scca
+
+        # School age group is 4 (one more than the maximum value in the age.group parameter)
+        school_age_group = 4
 
         # Make sure 1 year old child in Group 1
         adjusted_age = where(age == 1, 0.99, age)
```

Comment:
Would like to know if this is the right way to categorize 1 year old to Group 1.

Reply:
isn't group 1 kids under age 2? if so you don't need to do anything. i think it's just broken because thresholds and amounts are reversed in the parameter file

Reply:
Group 1 is infant(birth to 12 months). My understanding is the age <= 1.